cli/src/debugger.rs

```diff
@@ -613,8 +614,8 @@ impl<'p> CliData<'p> {
                 let pc = halted_state.program_counter;
                 if let Some(current_stack_frame) = halted_state.get_current_frame() {
                     println!(
-                        "Frame {}: {} @ {:#010x}",
-                        current_stack_frame, current_stack_frame.function_name, pc,
+                        "Frame {}: {} () @ {:#010x}",
+                        halted_state.current_frame, current_stack_frame.function_name, pc,
```

Comment:
Hmm this seems wrong. What am I missing? Seems like you are using the option?

Reply:
`halted_state.current_frame` is the index of the currently selected frame, which is what should be shown here.